CONTRIBUTING.md

```diff
@@ -14,11 +14,18 @@ standard library in the [Standard library developers Guide][std-dev-guide], comm
 
 ## About the [rustc-dev-guide]
 
-The [rustc-dev-guide] is meant to help document how rustc –the Rust compiler– works,
-as well as to help new contributors get involved in rustc development. It is recommended
-to read and understand the [rustc-dev-guide] before making a contribution. This guide
-talks about the different bots in the Rust ecosystem, the Rust development tools,
-bootstrapping, the compiler architecture, source code representation, and more.
+The [rustc-dev-guide] provides documentation on how the Rust compiler (rustc) works, 
```

Comment:
This seems unnecessarily passive. It could be rephrased to be active: as "documents" instead of "provides documentation on".